dev/archery/archery/cli.py

```diff
@@ -183,6 +183,8 @@ def _apply_options(cmd, options):
 @click.option("--with-r", default=None, type=BOOL,
               help="Build the Arrow R extensions. This is not a CMake option, "
               "it will toggle required options")
+@click.option("--with-gcs", default=None, type=BOOL,
```

Comment:
Can you keep this alphabetically ordered?

Reply:
yep.  done.

Reply:
It seems the option is duplicated now? (I'm curious that click doesn't complain about it)